auth_server/chart/docker-auth/README.md

```diff
@@ -0,0 +1,89 @@
+Helm Chart for docker_auth
+=======================
+
+## Introduction
+
+This [Helm](https://github.com/kubernetes/helm) chart installs [docker_auth](https://github.com/cesanta/docker_auth) in a Kubernetes cluster.
+
+## Prerequisites
+
+- Kubernetes cluster 1.10+
+- Helm 2.8.0+
+
+## Installation
+
+### Install the chart
+
+Install the docker-auth helm chart with a release name `my-release`:
+
+```bash
+# helm install --name=docker-auth docker-auth
```

Comment:
Get rid of the `#` here. I recognize that you are trying to visually distinguish the execution of the command and the results. But having it rendered as a comment is a bit confusing. You should just use two different blocks to make that distinction.  Or at a minimum, use the semi-recognized pattern of using a `$` to indicate the command.

```
$ helm install --name=docker-auth docker-auth
OUTPUT HERE
```

or

```
helm install --name=docker-auth docker-auth
```
```
OUTPUT HERE
```

Reply:
Done.